sky130-build/build.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+import argparse
+import copy
+import siliconcompiler
+import os
```

Comment:
I think all of these imports (except siliconcompiler) won't be necessary anymore.